src/main/java/com/icthh/xm/tmf/ms/communication/service/firebase/BuilderWrapper.java

```diff
@@ -0,0 +1,27 @@
+package com.icthh.xm.tmf.ms.communication.service.firebase;
+
+import com.google.firebase.messaging.AndroidConfig;
+import com.google.firebase.messaging.AndroidNotification;
+import com.google.firebase.messaging.ApnsConfig;
+import com.google.firebase.messaging.Aps;
+import com.google.firebase.messaging.MulticastMessage;
+import com.google.firebase.messaging.Notification;
+import com.google.firebase.messaging.WebpushConfig;
+import com.google.firebase.messaging.WebpushNotification;
+import lombok.Data;
+
+/**
+ * Wrapper around Firebase {@link MulticastMessage} builders.
+ */
+@Data
```

Comment:
do we really need @Data annotation here? Are you planning to use setters? If not consider @value annotation instead

Reply:
Done.